pkg/virt-controller/watch/clone/clone.go

```diff
@@ -259,6 +260,19 @@ func (ctrl *VMCloneController) syncSourceVMTargetVM(key string, source *k6tv1.Vi
 		ctrl.logAndRecord(vmClone, TargetVMCreated, fmt.Sprintf("created target VM %s for clone %s", targetVMInfo.Name, vmClone.Name))
 		syncInfo.targetVMCreated = true
 
+		targetVM, ok := targetVMObj.(*k6tv1.VirtualMachine)
+		if !ok {
+			return addErrorToSyncInfo(syncInfo, fmt.Errorf("VM object from vm informer cannot be converted to *VirtualMachine: %v", err))
+		}
+
+		if targetVM.Status.RestoreInProgress != nil {
+			logger.V(defaultVerbosityLevel).Infof("target VM %s is still not ready - restore %s still in progress", targetVM.Name, *targetVM.Status.RestoreInProgress)
```

Comment:
small thing is that in a previous log you write restore is ready to use and then here it will be a bit confusing that its written still in progress. maybe rephrase to status of restore not updated yet on the VM or something like that.

Reply:
done